<h3> Project Title </h3>
Evaluating Phylogenetic Tree Construction methods : Distance-based and Character-based

<h3> Team Members </h3>
Pushparaj Britto : 9178-5569

<h3> Abstract </h3>
A phylogenetic tree is a data structure shoving evolutionary relationships between biological species based upon similarities and differences in their physical or genetic characteristics. There are two major kinds of methods for generating phylogenetic trees: Distance-based (Phenetic) and Character-based (Cladistic). DNA sequences are applied for analyzing closely related species. On other hand, Protein sequences are used as input data to study wide range of taxa. I aim to find out the computational method best suited for generating phylogenetic tree using protein sequence as input.  
<h3> Plan of Action </h3>
  <p><b>Methods</b> 
  <br>There are two methods primarily used when adopting a Distance-based model. They are,
  <ul>
    <li>Neighbor-Joining</li>
    <li>UPGMA</li> 
  </ul>
  
  The widely used Character-based methods are as follows.
  <ul>
    <li>Maximum Likelihood</li>
    <li>Maximum Parsimony</li> 
  </ul>
 I have chosen Neighbor-Joining and Maximum Likelihood methods as respective Phenetic and Cladistic methods for evaluation and comparison.</p>
<p><b>Sequence</b><br> Phylogenetic trees can be constructued after processing either Protein or DNA sequences. I am going to use Protein sequences as my input.</p>
<p><b>Datasets</b><br> Protein Data Bank (RCSB PDB) will serve as my source for the input datasets.</p>
<p><b>Tools</b><br> I have identified PHYLIP 3.5 package as the primary tool to construct phylogenetic trees. It provides a wide-range of programs (PROTPARS, PROTDIST) to analyse sequence and generate distance matrix data as well as molecular sequence data. Tree constructing programs are too available in PHYLIP program library. T-Coffee can be used as the multpiple - sequence alignment tool</p>
<h3>References</h3>
<ol>
  <li>Guindon, S., & Gascuel, O. (2003, June 11). A Simple, Fast, and Accurate Algorithm to Estimate Large Phylogenies by Maximum Likelihood. In Laboratorie Informatique Robotique Micro√©lectronique Montpellier. September 23, 2014</li>
  <li>Joseph Felsenstein. Evolutionary trees from DNA sequences: A maximum likelihood approach.</li>
  <li>N Saitou M Nei. The neighbor-joining method: a new method for reconstructing phylogenetic trees.</li>
  <li>Kei Takahashi, Masatoshi Nei. Efficiencies of Fast Algorithms of Phylogenetic Inference Under the Criteria of Maximum Parsimony, Minimum Evolution, and Maximum Likelihood When a Large Number of Sequences Are Used.</li>
  <li>Kudipudi Srinivas, Allam Appa Rao, GR Sridhar, Srinubabu Gedela. Methodology for Phylogenetic Tree Construction.</li>
  <li>Mukul Subodh Bansal, Algorithms for efficient Phylogenetic Tree construction, Graduate Theses and Dissertations at Iowa State University, 2009</li>
</ol>
